task.go

```diff
@@ -116,6 +116,8 @@ func (r *marathonClient) KillApplicationTasks(id string, opts *KillApplicationTa
 //	opts:		KillTaskOpts request payload
 func (r *marathonClient) KillTask(taskID string, opts *KillTaskOpts) (*Task, error) {
 	appName := taskID[0:strings.LastIndex(taskID, ".")]
+	appName = strings.Replace(appName, "_", "/", -1)
```

Comment:
Sorry, I missed the comment in the related issue. Doesn't this mean that a call to `CreateApplication()` will fail as well if the given app ID is invalid?

Again, I'm not keen of getting this fixed right now as the issue and PR are about `KillTask`. Just want to make sure we document things we discover along the way.